src/translations/en.json

```diff
@@ -222,7 +222,7 @@
   "insuree.selectHeadInsuree.tooltip": "Select existing insuree as head",
   "insuree.addInsuree.alert.title": "Add insuree to family warning(s):",
   "insuree.addInsuree.alert.message": "However, insuree can still be added to the family.",
-  "insuree.InsureeNumberInput.error": "Invalid Insuree Number",
+  "insuree.InsureeNumberInput.error": "Insurance number already in use",
```

Comment:
Please remember about updating lokalise after changing labels.

Reply:
It is already updated.